It seems like `list(APPEND FEATURE_NET_OPTIONS -DCMAKE_REQUIRE_FIND_PACKAGE_WrapResolv:BOOL=ON)` triggers:

```
CMake Error at /mnt/vss/_work/1/s/scripts/buildsystems/vcpkg.cmake:857 (_find_package):
  Could not find a package configuration file provided by "WrapResolv" with
  any of the following names:

    WrapResolvConfig.cmake
    wrapresolv-config.cmake

  Add the installation prefix of "WrapResolv" to CMAKE_PREFIX_PATH or set
  "WrapResolv_DIR" to a directory containing one of the above files.  If
  "WrapResolv" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  cmake/QtFindPackageHelpers.cmake:107 (find_package)
  src/network/configure.cmake:15 (qt_find_package)
  cmake/QtFeature.cmake:678 (include)
  src/CMakeLists.txt:10 (qt_feature_evaluate_features)
```

--- a/ports/qtbase/portfile.cmake
+++ b/ports/qtbase/portfile.cmake
@@ -137,11 +137,14 @@ list(APPEND FEATURE_CORE_OPTIONS -DCMAKE_DISABLE_FIND_PACKAGE_WrapBacktrace:BOOL
     "openssl"             FEATURE_openssl
     "brotli"              FEATURE_brotli
     "securetransport"     FEATURE_securetransport
+    "dnslookup"           FEATURE_dnslookup
+    "dnslookup"           FEATURE_libresolv
     #"brotli"              CMAKE_REQUIRE_FIND_PACKAGE_WrapBrotli
     #"openssl"             CMAKE_REQUIRE_FIND_PACKAGE_WrapOpenSSL
  INVERTED_FEATURES
     "brotli"              CMAKE_DISABLE_FIND_PACKAGE_WrapBrotli
     "openssl"             CMAKE_DISABLE_FIND_PACKAGE_WrapOpenSSL
+    "dnslookup"           CMAKE_DISABLE_FIND_PACKAGE_WrapResolve
     )
 
 if("openssl" IN_LIST FEATURES)
@@ -152,7 +155,6 @@ endif()
 
 list(APPEND FEATURE_NET_OPTIONS -DCMAKE_DISABLE_FIND_PACKAGE_Libproxy:BOOL=ON)
 list(APPEND FEATURE_NET_OPTIONS -DCMAKE_DISABLE_FIND_PACKAGE_GSSAPI:BOOL=ON)
-list(APPEND FEATURE_NET_OPTIONS -DCMAKE_DISABLE_FIND_PACKAGE_WrapResolv:BOOL=ON)
 
 # Gui features:
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_GUI_OPTIONS

--- a/ports/qtbase/vcpkg.json
+++ b/ports/qtbase/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "qtbase",
   "version": "6.7.2",
+  "port-version": 1,
   "description": "Qt Base (Core, Gui, Widgets, Network, ...)",
   "homepage": "https://www.qt.io/",
   "license": null,
@@ -65,6 +66,7 @@
       "platform": "!(static & windows)"
     },
     "default-features",
+    "dnslookup",
     "doubleconversion",
     "freetype",
     "gui",
@@ -182,6 +184,18 @@
             "gles2"
           ],
           "platform": "android"
+        }
+      ]
+    },
+    "dnslookup": {
+      "description": "Enable DNS lookup support",
+      "dependencies": [
+        {
+          "name": "qtbase",
+          "default-features": false,
+          "features": [
+            "network"
+          ]
         }
       ]
     },

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7406,7 +7406,7 @@
     },
     "qtbase": {
       "baseline": "6.7.2",
-      "port-version": 0
+      "port-version": 1
     },
     "qtcharts": {
       "baseline": "6.7.2",

--- a/versions/q-/qtbase.json
+++ b/versions/q-/qtbase.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b8c493a18ed293e1418f464515c9f9ddf3dc1bfe",
+      "version": "6.7.2",
+      "port-version": 1
+    },
+    {
       "git-tree": "f8600119a4cecb137dd044bf72ab5a3c8b67ae02",
       "version": "6.7.2",
       "port-version": 0

